vaadin-ordered-layout-flow-parent/README.md

```diff
@@ -1,6 +1,6 @@
 # Layout components for Vaadin Flow
 
-This project is the Component wrapper implementation of [`<vaadin-ordered-layout>`](https://github.com/vaadin/vaadin-ordered-layout) element
+This project is the Component wrapper implementation of [Vaadin ordered layout web components](https://github.com/vaadin/web-components/) (HorizontalLayout, VerticalLayout etc.)
```

Comment:
How about a comma-separated listing of the individual elements, each as a respective link to its package?